src/fhir/api.ts

```diff
@@ -0,0 +1,31 @@
+import {patientSchema} from "../components/reports/formDataValidation";
+import {createPatientEntry} from "./resources";
+
+/**
+ * Create a report bundle
+ * @param patient FHIR patient entity
+ */
+export const bundleRequest = (patient: typeof patientSchema) => {
+  return {
+    url: "/",
+    method: "POST",
+    headers: {'Content-Type': 'application/fhir+json;charset=UTF-8'},
+    body: JSON.stringify(createBundle(patient)),
+  };
+}
+
+export const createBundle = (patient: typeof patientSchema) => {
+  const mrn = patient?.identifier?.at(0)?.value;
```

Comment:
What happens if we don't have a patient here? Does the error eventually get caught in your `catch` block in `onSuccessfulSubmitHandler`?